<?xml version="1.0"?>

<launch>
  
  <!-- SINGABOAT-VRX Mission Manager -->
  <node pkg="singaboat_vrx" name="singaboat_mission_manager" type="singaboat_mission_manager.py" output="screen"/>

  <!-- SINGABOAT-VRX Scene Perception -->
  <node pkg="singaboat_vrx" type="singaboat_scene_perception.py" name="singaboat_scene_perception" output="screen"/>

  <!-- SINGABOAT-VRX Gymkhana Challenge -->
  <node pkg="singaboat_vrx" type="singaboat_gymkhana_challenge.py" name="singaboat_gymkhana_challenge" output="screen"/>
    
  <!-- SINGABOAT-VRX Inverse Kinematics -->
  <node pkg="singaboat_vrx" type="singaboat_inverse_kinematics.py" name="singaboat_inverse_kinematics" output="screen"/>

</launch>
